solr/prometheus-exporter/src/java/org/apache/solr/prometheus/exporter/SolrExporter.java

```diff
@@ -103,6 +104,13 @@ public class SolrExporter {
   private static final String ARG_CREDENTIALS_HELP =
       "Specify the credentials in the format username:password. Example: --credentials solr:SolrRocks";
 
+  private static final String[] ARG_SSL_FLAGS = {"-ssl", "--ssl-enabled"};
+  private static final String ARG_SSL_METAVAR = "SSL_ENABLED";
+  private static final String ARG_SSL_DEST = "ssl_enabled";
+  private static final boolean ARG_SSL_DEFAULT = false;
+  private static final String ARG_SSL_HELP =
+      "Enable tls to Solr. Expects following env varables: SOLR_SSL_KEY_STORE, SOLR_SSL_KEY_STORE_PASSWORD, SOLR_SSL_TRUST_STORE, SOLR_SSL_TRUST_STORE_PASSWORD. Example: --ssl-enabled true";
```

Comment:
is TLS normally capitalized?   "Enable TLS connection to Solr.".

Reply:
Good point. I have updated the text with your suggestion and also fixed the typo in "varables".